### ISSUE TITLE
Время придумать джоб обжективы

### ISSUE THREAD

**Issue by Signmark** [ITBlackwood/ATMTA-Paradise#75]
http://pastebin.com/59BQiUbh

У меня уже есть довольно обширный списочек, но я прошу МОАР ИДЕЙ
Обжективы для профессий которые надо выполнить к концу раунда.

**Comment by Zurro13** [comment]
Придумать можно много всего, к примеру для той же инженерки обеспечить в сети не менее 600кВт, построить работающий ТЭГ, да запитать те же локации на недостроенной станции. Для атмоса обеспечить запасы (давление в станционных баках) какого-либо газа не менее х кПа (не меньше, чем там на начало смены). Да и шахтерам элементарно иметь х поинтов на конец смены. Вопрос в том, что из этого получится проверять в автоматическом режиме, а что будет требовать ручного контроля.

**Comment by ITBlackwood** [comment]
Нет, придумать действительно можно много чего. Проблема в том, как расписать это в коде так, чтобы игра понимала, что обджектив выполнен.

**Comment by Zurro13** [comment]
Ну вот собственно из того, что я расписал:
Раз игра как-то проверяет, запитана ли вся станция, значит должна быть возможность проверять запитку отдельных зон.
Проверить напряжение в сети ... пока с повернетами не до конца ясно, но теоретически возможно, мне так кажется.
Построить работающий ТЭГ - как пример того, к чему нужно пояснение, имеет ли смысл задумывать какие-то новые постройки.
Давление в баках - показания с сенсоров снимаются, значит теоретически проверка возможна.
Количество поинтов на карточке шахтера тоже.

Собственно эти варианты и я привел в качестве примера, для того, чтобы получить ответ на вопрос, какие из них получится реализовать (и имеет смысл придумывать аналогичные), а какие нет/очень сложно, и нету смысла даже заморачиваться.

**Comment by Signmark** [comment]
Игра ведь уже проверяет многие вещи в конце, как например проблемы с энергией, неубранный мусор, еще что-то там
Вот к ним я подрублюсь в первую очередь

**Comment by Zurro13** [comment]
Поскольку я не зарегистрирован вконтакте, напишу здесь:
РД: (так как РД в первую очередь администратор)
Обеспечить эвакуацию всех ученых, или их тел.
Наличие электропитания на аутпосте (в принципе актуально только при длинных раундах)
Наличие живых слизней в количестве х штук
Отправка всех технологий, кроме illegal, уровня не меньше 3, на каргошаттле
Исследование животных (если возможно проверить наличие в ксенобио конкретной твари с астероида)
Обеспечить улучшение какого-то конкретного типа техники на станции (опять же, вопрос в проверке)
Обеспечить улучшение научного поста ... к примеру, крио там смонтировать или химраздатчик установить.
Наличие не менее чем одной обезьяны с мутациями вида х
Отправка не менее чем х вариантов мутаций на шаттле.

Ученый:(если есть возможность выдавать задачу в зависимости от специализации, то будет логичнеее)
Отправка какой-то конкретной ветки технологий уровня не меньше х
Отправка ядер слизней конкретного вида, в количестве не меньше х
Отправка каких-то сложных химикатов в количестве не меньше х
Отправка исследованных strange object, в количестве не меньше х

Роботех:
Собрать х конкретных ботов (медботы, клинботы и тому подобные)
Собрать х мехов
Изготовить х (комплектов) протезов для медбея
Поддерживать целостность всех изготовленных мехов на уровне не менее х% (ХЗ можно ли проверить)
Отправить меха на шаттле в комплектации (к примеру рипли с клешней, рцд, огнетущителем и кабелеукладчиком)(ХЗ что с проверкой)
Отправить х готовых неактивных боргов на шаттле
